x-pack/platform/plugins/private/file_upload/public/importer/routes.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  IndicesIndexSettings,
+  MappingTypeMapping,
+} from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import {
```

Comment:
We can use import type here and above

Reply:
Good spot, I'm too used to the `consistent-type-imports` eslint rule we have in our other plugins.
I think I'll add that rule to the `file_upload` plugin too in a separate PR.

Reply:
https://github.com/elastic/kibana/pull/212044